code/src/components/LoadingStuff.css

```diff
@@ -0,0 +1,168 @@
+.circle-1 {
```

Comment:
Love the name "Loading stuff"!! Also, it looks perfect, very in keeping with your theme